src/databases/functions/users/set-w-pu-user.js

```diff
@@ -0,0 +1,13 @@
+const {User} = require('../../models');
+
+module.exports = async (userId, score) => {
+  if (!userId || !score) return null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `setWPUUser` should throw an error when the required parameters `userId` and `score` are not provided. Returning `null` can make it harder to debug issues as it doesn't provide any information about what went wrong.
```diff
-  if (!userId || !score) return null;
+  if (!userId || !score) throw new Error('UserId and score are required parameters');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->